ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/Examples.hs

```diff
@@ -143,6 +149,30 @@ instance Inject Examples where
         => Proxy f -> Labelled a -> Labelled b
       inj p = fmap (fmap (inject' p startBounds idx))
 
+newtype WrapLedgerTables blk = WrapLedgerTables ( LedgerTables (ExtLedgerState blk) ValuesMK )
+
+instance Inject WrapLedgerTables where
+   -- TODO: here the only options I see are either to remove the signature of
```

Comment:
Maybe if you define the method implementation as its own top-level `injectWrapLedgerTables` definition, then you could pare down its signature to exactly what GHC suggests, instead of having to satisfy `-XInstanceSigs`.

```
instance Inject WrapLedgerTables where
  inject = injectWrapLedgerTables
  
injectWrapLedgerTables :: ...
```

Reply:
I should have thought about that :facepalm: Thanks!